### PR TITLE
Signing locally the transactions

### PR DESCRIPTION
**Pull request by jcortejoso**
## Description
Signing locally the transactions

### CODE REVIEW

File: src/controllers/neverminedFaucet.js [hunk 1]
```diff
@@ -96,17 +97,28 @@ const NeverminedFaucet = {
...
+                        networkId: networkId,
+                        chainId: networkId
+                    }
+                }
             },
-            config.server.faucetPassword
+            config.server.privateKey
         )
-        return hash
+        logger.info(`SignedData: ${JSON.stringify(signedData, null, 2)}`)
```
Comment (aaitor) [inline src/controllers/neverminedFaucet.js:115]: logger.debug?

File: src/controllers/neverminedFaucet.js [hunk 2]
```diff
@@ -96,17 +97,28 @@ const NeverminedFaucet = {
...
+        web3.eth.sendSignedTransaction(signedData.rawTransaction)
+            .once("confirmation", function() {
+                console.log('*** Transaction Confirmed ***')
```
Comment (aaitor) [inline src/controllers/neverminedFaucet.js:118]: logger.debug

File: src/controllers/neverminedFaucet.js [hunk 3]
```diff
@@ -96,17 +97,28 @@ const NeverminedFaucet = {
...
+            })
+            .on("error", console.error)
```
Comment (aaitor) [inline src/controllers/neverminedFaucet.js:120]: logger.error

**Review by aaitor — CHANGES_REQUESTED** [review]

**Review by aaitor — APPROVED** [review]

**Review by eruizgar91 — APPROVED** [review]
:rocket:

**Review by H34D — APPROVED** [review]
LGTM

**Pull request merged**